packages/turf-helpers/index.js

```diff
@@ -331,5 +349,6 @@ module.exports = {
     multiPolygon: multiPolygon,
     radiansToDistance: radiansToDistance,
     distanceToRadians: distanceToRadians,
-    distanceToDegrees: distanceToDegrees
+    distanceToDegrees: distanceToDegrees,
+    bearingToAngle: bearingToAngle
```

Comment:
👍 Just a side note: This syntax "looks ugly" since now you can use **Shorthand property names in ES2015**. https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer
However this is needed because TurfJS aims to be ES5 compatible.